ci/docker/run/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:18.04
+
+RUN apt-get update \
+    && apt-get install --no-install-recommends -y \
+                        git=1:2.17.1-1ubuntu0.5 \
+                        build-essential=12.4ubuntu1 \
+                        autoconf=2.69-11 \
+                        flex=2.6.4-6 \
+                        bison=2:3.0.4.dfsg-1build1 \
+                        bc=1.07.1-2 \
+                        perl=5.26.1-6ubuntu0.3 \
+                        python3=3.6.7-1~18.04 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+ARG REPO=https://github.com/verilator/verilator
+ARG SOURCE_COMMIT=master
+
+WORKDIR /tmp
+
+RUN git clone ${REPO} verilator && \
+    cd verilator && \
+    git checkout ${SOURCE_COMMIT} && \
+    autoconf && \
+    ./configure && \
+    make -j `nproc` && \
+    make install && \
+    cd .. && \
+    rm -r verilator
+
+ADD verilator-wrap.sh /usr/local/bin/verilator-wrap.sh
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use COPY instead of ADD for files and folders](https://app.codacy.com/manual/wsnyder/verilator/pullRequest?prid=4756540)